subjects/file-struct/README.md

```diff
@@ -0,0 +1,46 @@
+## File struct
+
+In order to create new directories or folders in Linux or Unix-like operating systems you need to use the `mkdir` command. `mkdir` stands for “make directory.” you can use it like so:
+
+```console
+User-> mkdir my_folder
+User-> ls
```

Comment:
it will be better if you give the student some information about `ls` commands

Reply:
Will be given in the strange file exercise because ls appears there